packages/date-picker/src/vaadin-infinite-scroller.js

```diff
@@ -201,6 +201,21 @@ export class InfiniteScroller extends HTMLElement {
     }
   }
 
+  /** @protected */
+  disconnectedCallback() {
+    if (this._debouncerScrollFinish) {
+      this._debouncerScrollFinish.cancel();
+    }
+
+    if (this._debouncerUpdateClones) {
+      this._debouncerUpdateClones.cancel();
+    }
+
+    if (this.__pendingFinishInit) {
```

Comment:
**note:** I saw that some tests finish before the `_finishInit` handler is executed. In such cases, we need to cancel its execution to prevent `performUpdate` from throwing an exception.

Reply:
IMO we should also consider updating these tests to wait for overlay open.